.ort.yml

```diff
@@ -73,10 +73,10 @@ excludes:
     reason : "TEST_OF"
     comment: "Licenses contained in this directory are used for testing and do not apply to the OSS Review Toolkit."
   scopes:
-  - name   : "devDependencies"
+  - pattern   : "devDependencies"
```

Comment:
Please align the colon here and below.

Reply:
There are several further unaligned colons. I'd like to keep this out of scope of this change, in particular since it is fixed in the following commit anyway.

Reply:
You are breaking alignment of these formerly aligned blocks in this commit, so you can hardly say it's "out of scope", but as it's undone in the next commit anyway, ok.

Reply:
I agree only partially, because the alignment was inconsistent and thus broken already before on file scope, see the license finding curations. It was consistent only in the narrower scope of path excludes.